api/README.md

```diff
@@ -67,6 +80,29 @@ If you do not have `swagger-cli` installed, you can install it with the followin
 npm install -g swagger-cli
 ```
 
+## Usage - Production
+
+Build a Python "wheel" and install it in a production environment. From the `api/` directory run the following commands:
```

Comment:
WSGI don't technically require a python wheel, they just require the module to be installed on the system / virtual environment. pip will often use wheels during intermediate build steps, and precompiled wheels can be useful for installing the modules on other machines, but a wheel is not strictly required by tools like `gunicorn`. 

I think I would replace with a bit more detail:

```
A WSGI server is needed to run our application. A WSGI (Web Server Gateway Interface) server is a bridge between a web server and web applications. The server handles requests, invokes the web application, translates responses back to HTTP, and also manages concurrency to ensure the server can handle multiple requests simultaneiously.

While Flask does provide a built-in development server, it is not intended for production use. Therefore, we utilize a third-party, production-grade WSGI server to manage our application. Various options exist, but we've chosen `gunicorn`. 

WSGI servers like `gunicorn` require the python module to be installed on the system, in order to properly import the application class. To run the server, use pip to install `openapi_server` and then start the application from `gunicorn`.
```

Reply:
This is really good. I'll put it in. Thank you!

Reply:
Added in [c6ec31d](https://github.com/hackforla/HomeUniteUs/pull/544/commits/c6ec31df74408ba7d0736fe449b79ad16649bc9f). Thanks!